strawberry/extensions/tracing/datadog.py

```diff
@@ -45,34 +78,38 @@ def on_operation(self) -> Iterator[None]:
             f"{self._operation_name}" if self._operation_name else "Anonymous Query"
         )
 
-        self.request_span = tracer.trace(
+        self.request_span = self._create_span(
+            LifeCycleStep.REQUEST,
```

Comment:
Note that this is not really a request it is operation, one request can have many operations. especially if you use websockets and your connection is alive for as long as the client might query for something.

Reply:
ah, yes, good point 😊

Reply:
Renamed this to OPERATION.